core/server/data/migrations/versions/4.0/25-populate-members-paid-subscription-events-table.js

```diff
@@ -29,6 +29,14 @@ module.exports = createTransactionalMigration(
             if (interval === 'month') {
                 return amount;
             }
+
+            if (interval === 'week') {
+                return amount * 4;
+            }
+
+            if (interval === 'day') {
+                return amount * 30;
```

Comment:
Are the `30` and `4` just above too general to "the average week or month" or doesn't it really matter for us?

Reply:
I spoke with Matt and he's happy with these - "Yep, they’re edge cases anyway"